.github/workflows/main.yml

```diff
@@ -42,7 +42,7 @@ jobs:
         # the framework string. As a result "vue" will map
         # to both "vue" and "vue-vite" apps. Similarly,
         # "react" will map to both "react" and "react-vite" apps.
```

Comment:
This comment is outdated

Reply:
Fixed in https://github.com/ionic-team/starters/pull/1795/commits/eab0509ac645d0cfcb3bebed2b21ec7bf54c6db0